holmes/plugins/toolsets/confluence/confluence.py

```
@@ -0,0 +1,376 @@
+                    type="integer",
+                    required=False,
+                ),
+                "type": ToolParameter(
+                    description="Filter by space type: 'global' or 'personal'",
+                    type="string",
+                    required=False,
+                ),
+            },
+        )
+
+    def _invoke(self, params: dict, context: ToolInvokeContext) -> StructuredToolResult:
```

Ruff (0.14.13)

Unused method argument: `context`

(ARG002)